--- a/DESCRIPTION
+++ b/DESCRIPTION
@@ -28,9 +28,8 @@
 BugReports: https://github.com/tidyverse/dplyr/issues
 Depends: R (>= 3.3.0)
 Imports: ellipsis, generics, glue (>= 1.3.2), lifecycle (>= 1.0.0),
-        magrittr (>= 1.5), methods, R6, rlang (>= 0.4.10), tibble (>=
-        2.1.3), tidyselect (>= 1.1.0), utils, vctrs (>= 0.3.5), pillar
-        (>= 1.5.1)
+        magrittr (>= 1.5), methods, R6, rlang (>= 0.4.10),
+        tidyselect (>= 1.1.0), utils, vctrs (>= 0.3.5), pillar (>= 1.5.1)
 Suggests: bench, broom, callr, covr, DBI, dbplyr (>= 1.4.3), knitr,
         Lahman, lobstr, microbenchmark, nycflights13, purrr, rmarkdown,
         RMySQL, RPostgreSQL, RSQLite, testthat (>= 3.0.2), tidyr, withr
